### ISSUE TITLE
Repanse's "The Sword of Lyonesse" Quest Battle does not work (mod on steam)

### ISSUE THREAD

**Issue by mjones1052** [chadvandy/cbfm_wh3#78]
One of the units doesn't spawn it seems and you can't complete it. If you kill everything you just run around forever. I didn't see it listed here or on the fixed page so figure I would submit it.

**Comment by Grithok** [comment]
Just ran into this as well.

**Comment by chadvandy** [comment]
Unable to verify this myself, played through fine. The reinforcement army in question seems to trigger whenever any of the three TK armies is brought to 40% health, or in 4 minutes, whichever of those conditions happens first. Seems to be set up fine, not sure why it wouldn't fire for you all.
```
gb:message_on_time_offset("reinforce", 240000,"01_intro_cutscene_end");	--might be a little quick tbd
--gb:message_on_time_offset("scorpion_ambush", 240000,"01_intro_cutscene_end");

ga_tombk_1:message_on_casualties("tombk_1_dying", 0.6);
--ga_tombk_1:message_on_commander_dead_or_routing("tombk_1_dead", 1);
ga_tombk_2:message_on_casualties("tombk_2_dying", 0.6);
ga_tombk_3:message_on_casualties("tombk_3_dying", 0.6);

ga_tombk_4_reinforce:reinforce_on_message("reinforce", 10);
--ga_tombk_4_reinforce:reinforce_on_message("tombk_1_dying" and "tombk_2_dying" or "tombk_3_dying", 10);	--this probably doesn't work, intent is to have reinforcements trigger when any 2 out of the 3 armies are dying
ga_tombk_4_reinforce:reinforce_on_message("tombk_1_dying", 10);
ga_tombk_4_reinforce:reinforce_on_message("tombk_2_dying", 10);
ga_tombk_4_reinforce:reinforce_on_message("tombk_3_dying", 10);
```

**Comment by mjones1052** [comment]
It never triggered. I ran around the entire empty map after killing everything else.


Regards,
________________________________

**Comment by Harmandodelavega** [comment]
https://steamcommunity.com/sharedfiles/filedetails/?id=2860157745 this guy's got a workaround

**Comment by mfvreeland** [comment]
Fixed by CA in 2.2
